R/RLearner_FDAclassif_knn.R

```diff
@@ -0,0 +1,42 @@
+#' @export
+makeRLearner.fdaclassif.knn = function() {
+  makeRLearnerClassif(
+    cl = "fdaclassif.knn",
+    package = "fda.usc",
+    par.set = makeParamSet(
+      makeIntegerLearnerParam(id = "knn", lower = 1L, default = NULL, special.vals = list(NULL)),
+      makeDiscreteLearnerParam(id = "metric", default = "metric.lp", values = c("metric.lp", "metric.kl",
+        "metric.hausdorff", "metric.dist")),
+      makeDiscreteLearnerParam(id = "type.CV", default = "GCV.S", values = c("GCV.S", "CV.S", "GCCV.S")),
+      # trim and draw (= plot!) are the par.CV parameters
+      makeNumericLearnerParam(id = "trim", lower = 0L, upper = 1L, default = 0L),
+      makeLogicalLearnerParam(id = "draw", default = FALSE, tunable = FALSE)
+    ),
+    par.vals = list(draw = FALSE),
+    properties = c("twoclass", "multiclass", "numerics"),
+    name = "Knn on FDA",
+    short.name = "knnFDA",
+    note = "Draw parameter is set to FALSE as default."
```

Comment:
Why is this in the note? If it's different from what the learner itself does, it should be set in par.vals only and in the parameter definition to what the learner does.